src/components/Accordion/accordion.scss

```diff
@@ -41,10 +43,9 @@
 
   &__panel {
     overflow: hidden;
+    transition: max-height $accordion-animation-duration $accordion-animation-easing;
 
-    &[ aria-hidden='true' ] {
-      max-height: 0;
-    }
+    max-height: 0;
```

Comment:
this is the only change you have to do to get the accordion to work. 
the problem was, that the `accordion__panel` had a `max-height` of `auto` as standard. it needs the `max-height` of `0`.

Reply:
so we can say, you can revert everything else 👍 😄

Reply:
No, still better solution now, without anime.js and therefore slimmer.

Reply:
true that, but we decided once to include animejs to show a example in javascript. but anyways i will accept this.